source/core/coverage/ut_coverage_helper.pkb

```diff
@@ -0,0 +1,109 @@
+create or replace package body ut_coverage_helper is
+  /*
+  utPLSQL - Version X.X.X.X
+  Copyright 2016 - 2017 utPLSQL Project
+
+  Licensed under the Apache License, Version 2.0 (the "License"):
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+  */
+
+  g_coverage_id integer;
+  g_develop_mode boolean := false;
+
+  function get_coverage_id return integer is
+  begin
+    return g_coverage_id;
+  end;
+
+  function  is_develop_mode return boolean is
+  begin
+    return g_develop_mode;
+  end;
+
+  function coverage_start(a_run_comment varchar2) return integer is
+    pragma autonomous_transaction;
+  begin
+    --those are Global Temporary tables for profiler usage only
+    delete from plsql_profiler_data;
```

Comment:
After a rethink we need a `union`.
If running in `utplsql_develop` mode, we will exclude Unit Test objects from coverage reporting additionally we will exclude all objects that are part of utplsql framework from the coverage.

Added a comment to indicate that.